Birthday-2024-Project/ScenePrefabs/gallery.tscn

```diff
@@ -0,0 +1,370 @@
+[gd_scene load_steps=31 format=3 uid="uid://k787vo3otomq"]
```

Comment:
minor thing: change the name of the scene to "Gallery" to have it consistent with other scenes